src/Snak/PropertyValueSnak.php

```diff
@@ -18,7 +18,10 @@
  */
 class PropertyValueSnak extends SnakObject {
 
-	private $dataValue;
+	/**
+	 * @since 3.1
```

Comment:
This was protected before 1.0. Just something I find funny, not worth documentation.